src/index.ts

```diff
@@ -33,6 +36,16 @@ process.on('unhandledRejection', (reason) => {
   }
 });
 
+function cleanUpTempStorage() {
+  const { IMAGE_STORAGE_ROOT } = config;
+  try {
+    emptyDirSync(IMAGE_STORAGE_ROOT);
```

Comment:
Customer has complained that sometimes storage isnt emptied properly. This is likely to be an even bigger concern if we inherited a PV used elsewhere, so we now ensure it's empty upon startup